app/lib/bk/compat/parsers/circleci/docker.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
-require_relative '../../pipeline/plugin'
-require_relative '../../pipeline/step'
+require_relative '../../models/plugin'
```

Comment:
You were missing the circle-ci specific step class to require (remember to adapt the change if the file is moved as suggested):

```suggestion
require_relative '../../models/plugin'
require_relative '../../models/circleci/circleci_step'

```

Reply:
Adjusted - also second add becomes `../../models/steps/circleci` with the above recent change 👍